### PR TITLE
Karma Store page

### PR DESCRIPTION
**Pull request by DelroyGayle**
<!--

London class 8 - Delroy Gayle - HTML CSS Coursework - Week 2

# Notes

- What did you find easy?

Linking one page to another

- What did you find hard?
Practically everything especially formatting and aligning the 'order form'

- What do you still not understand?

Align-items/content in regards to Flexbox

- Any other notes?
 
Would have like more information regarding CSS Forms. The class subject was too brief. Could have done with more detail.

### CODE REVIEW

**Comment by DelroyGayle** [comment]
Finally completed the changes

**Review by amellie — COMMENTED** [review]
Great effort, Delroy. Please keep it up! Here are some of my comments

File: css/style.css [hunk 1]
```diff
@@ -413,3 +407,118 @@ input:placeholder-shown{
...
+ /* However do NOT colour the text!
+  
+ label > input[type="radio"]:checked + * {
+  color: #ff6200;
+ }
+  
+ */
+
+ 
+ /* First step: hide the unstyleable checkbox */
```
Comment (mandeepsangha) [inline css/style.css:453]: I like how your comments explain what you are doing, makes following along very easy

File: store.html [hunk 2]
```diff
@@ -86,9 +86,15 @@ <h1 class="order-heading">Order your Karma wifi<br/>device today!</h1>
                                     <!-- Create a placeholder -->
                                     <option value="" disabled selected hidden><em>Select your city...</em></option>
```
Comment (mandeepsangha) [inline store.html:88]: Really nice use of <em> here, i ending up using CSS here as I could not get <em> to work.

File: css/style.css [hunk 3]
```diff
@@ -379,17 +375,15 @@ blockquote::after {
...
 
 a.terms-conds:link {
-  color: red;
+  color: #ff6200;
 }
 
 a.terms-conds:visited {
-  color: red;
+  color: #ff6200;
 }
```
Comment (mandeepsangha) [inline css/style.css:385]: a few big spaces in the CSS here, maybe close the gaps or add comments

File: css/style.css [hunk 4]
```diff
@@ -379,17 +375,15 @@ blockquote::after {
...
 
 
 
-
-
 input[type=text] {
   padding:10px;
   margin:10px 0 20px 0;
```
Comment (mandeepsangha) [inline css/style.css:389]: i prefer to use rem rather than px

**Review by mandeepsangha — COMMENTED** [review]
Hi Delroy, i have completed your review. 

- I really liked your use of comments, made it easy to follow what you were doing. 
- Really interesting reading your code, gave me some thoughts about things I can implement in my code
- Your City <em> tags worked! is really nice to see that em tags can work, i ended up using CSS to solve this

File: css/style.css [hunk 5]
```diff
@@ -6,14 +6,519 @@ body {
...
+    align-items: center;
+    justify-content: space-between;
+    height: 5rem;
+    margin-bottom: 1rem;
+    padding-top: 1rem;
+}
+  
+ 
+.karma-logo {
+    width: 2rem; /* if you set NO width, the image will not show! */
```
Comment (MarinaAlmeida20) [inline css/style.css:29]: nice comments here

File: css/style.css [hunk 6]
```diff
@@ -6,14 +6,519 @@ body {
...
+
+   /* always show 'Meet Karma' in black */
+.meet-Karma {
+      color: black;
+      font-weight:500;
+ }  
+
+   /* FLEXBOX for the LOGO & Navigation Bar */
+
+.LogoNavbox {
```
Comment (MarinaAlmeida20) [inline css/style.css:77]: Try to keep the first letter of the class small

File: index.html [hunk 7]
```diff
@@ -8,12 +8,118 @@
...
+   
+     </main>
+
+     <hr class="drawaline">
+
+     <footer class="thefooter">
+        <p class="join-us">Join us on</p>
+        <figure class="SocialMediaFlexBox">
+          <a href="#">
+                <img src="img\twitter-icon.svg" alt="Twitter link"; height=20px; width=20px; class="SocialMedia">
```
Comment (MarinaAlmeida20) [inline index.html:105]: avoid using height and width in HTML files.

**Comment by MarinaAlmeida20** [comment]
Hi Delroy, I finished my review here, you did nice work here on every page you will become better. keep going.

**Review by MarinaAlmeida20 — APPROVED** [review]

**Pull request closed without merging**